ddp-workspace/projects/ddp-atcp/src/app/components/statistics/statistics.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { CompositeDisposable, NGXTranslateService } from 'ddp-sdk';
+
+interface CountersI {
+  [key: number]: {
+    Number: string;
+    Text: string;
+  };
+}
+
+interface CountersWithEmojiI {
+  [key: number]: {
+    Width: string;
+    Emoji: string;
+    Name: string;
+  };
+}
+
+@Component({
+  selector: 'app-statistics',
+  templateUrl: './statistics.html',
+  styleUrls: ['./statistics.scss']
+})
+export class StatisticsComponent implements OnInit, OnDestroy {
+  public countersFirst: CountersI = [];
+  public countersSecond: CountersI = [];
+  public countersThird: CountersWithEmojiI = [];
+  private anchor = new CompositeDisposable();
+  constructor(private ngxTranslate: NGXTranslateService) {
+  }
+
+  public ngOnInit(): void {
+    const translate$ = this.ngxTranslate.getTranslation([
```

Comment:
we have a comment from April 22th:

> Also, could you please specify what data type returns this.ngxTranslate.getTranslation() method? There can be string or object, this will be enough.